WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
De jure titles aren't connected to the Christian church. They are supposed to be the structures that are the defaults that you and the AI take. There is a major reason why the Vanilla rendition does not limit them.

It's not about the player adapting to them, but rather, the AI. The AI is not smart enough to figure out that it can't create certain de jure titles, nor is it capable of going around that. The restrictions for de jure titles are one of the biggest mistakes in the Mod.